src/NuGet.Core/NuGet.Common/Logging/NullLogger.cs

```diff
@@ -8,7 +8,7 @@ namespace NuGet.Common
 {
     public class NullLogger : LoggerBase
     {
-        private static ILogger _instance;
+        private static ILogger? _instance;
 
         public static ILogger Instance
```

Comment:
What optimization? I was annotating exactly what was implemented

Reply:
the lazy creation of the null logger. It's not really heavy so no need to delay it.
My bad, I should've pointed out that it was the existing code. 

Up to you whether you change it or not.

Reply:
ah, since I had to annotate hundreds of APIs, I didn't pay attention to most of what I was annotating. Trying to resolve all the compiler errors, and minimize risk of regression by avoiding unnecessary code changes.